aws-lambda/aws-lambda.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for AWS Lambda
+// Project: http://docs.aws.amazon.com/lambda
+// Definitions by: Michael Skarum <https://github.com/skarum>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "aws-lambda" {
+
+    export interface Records {
+        Records: Record[];
+    }
+    interface Record {
+        EventVersion: string;
+        EventSubscriptionArn: string;
+        EnventSource: string;
+        Sns: SNS;
+        kinesis: Kinesis;
+    }
+    interface SNS {
+        Type: string;
+        MessageId: string;
+        TopicArn: string;
+        Subject: string;
+        Message: string;
+        Timestamp: Date;
+    }
+
+    interface Kinesis {
+        data: string;
+    }
+
+    export interface Context {
+        log(message: string, object: any): void;
+        fail(message: string): void;
+        succeed(message: string): void;
+        succeed(object: any): void;
+        succeed(message: string, object: any): void;
+        awsRequestId: string;
+        getRemainingTimeInMillis(): number;
+    }
+
+
+    export function Callback(error?: any, message?: string): void;
```

Comment:
looks like this should be

```ts
export type Callback = (error?: any, message?: string) => void;
```

since there's no actual function Callback exported.